python_otbr_api/tlv_parser.py

```diff
@@ -50,6 +50,7 @@ class MeshcopTLVType(IntEnum):
     PERIOD = 55
     SCAN_DURATION = 56
     ENERGY_LIST = 57
+    VENDOR_UNKNOWN = 74
```

Comment:
Can we give this a better name or add a comment explaining where we've seen this?

Reply:
I had `APPLE_UNKNOWN` first, but sounded strange too :sweat_smile:  Do you have a good idea?

Maybe:

```suggestion
    # Seen in a dataset imported through iOS companion app
    APPLE_TAG_UNKNOWN = 74
```

Reply:
I think `APPLE_TAG_UNKNOWN` is fine 👍